homeassistant/components/humidity/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "domain": "humidity",
```

Comment:
The name humidity is too broad. It doesn't show what kind of thing we're integrating. It could be anything that has some relation to humidity. I propose we change to the name of a device that controls humidity, eg humidifier, since that's what this entity integration should represent. I think it's ok to include dehumidifiers under this name, if that interface is similar enough.

Reply:
done